cmd/eval.go

```diff
@@ -13,7 +13,7 @@ import (
 	comboErrors "github.com/operator-framework/combo/pkg/errors"
 	"github.com/operator-framework/combo/pkg/template"
 	"github.com/spf13/cobra"
-	"gopkg.in/yaml.v3"
+	"gopkg.in/yaml.v2"
```

Comment:
Oh I just noticed this downgrade, any idea what caused this?

Reply:
Not entirely sure, I'll change it back and make sure it isn't causing issues. I think it could've been changed when I was playing around with Cobra the other day.